### ISSUE TITLE
Code Explorer needs additional icons for VB6 DeclarationTypes

### ISSUE THREAD

**Issue by mansellan** [rubberduck-vba/Rubberduck#3950]
Currently, parsing a VBForm causes an exception here: https://github.com/rubberduck-vba/Rubberduck/blob/14743dc32ef02938b569b841fe94e8d250217b72/Rubberduck.Core/Navigation/CodeExplorer/CodeExplorerComponentViewModel.cs#L44

as the declaration type for VBForms is not in the dictionary. We should identify the missing declaration types and create icons for them. It may be worthwhile to have a default too (or at least guarding with ContainsKey), to avoid blowing up CE if anything unexpected is found.

**Comment by mansellan** [comment]
So, I'd like to add icons for some of the 'missing' values of the [ComponentType](https://github.com/rubberduck-vba/Rubberduck/blob/next/Rubberduck.VBEEditor/SafeComWrappers/VB/Enums/ComponentType.cs) enum. Values 4 through 10 are only in the VB6 library:

4: ResFile
5: VBForm (we should just use the existing icon here)
6: MDIForm
7: PropPage
8: UserControl
9: DocObject
10: RelatedDocument

11: ActiveXDesigner - Exists in VBA but has no icon. Perhaps only applicable to 2000\XP dev editions and VB6.

I've picked out some nice ones from Fugue and SharpDevelop, but the trouble is that we're nearly out of flags on [DeclarationType](https://github.com/rubberduck-vba/Rubberduck/blob/next/Rubberduck.Parsing/Symbols/DeclarationType.cs), which we use as the icon key. I tried changing it to base on long, and it didn't seem to break any tests, but I'm aware its a hugely important enum, so messing with it may be a non-starter.

If this isn't gonna fly, any other ideas?
